CSharpOOP2/Car.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace Kristina_Kulich__Application.CSharpOOP2;
+
+public class Car
+{
+    public string Model;
+    public int ProductionYear = 2020;
+    public bool Diesel;
+    public double FillLevel;
+
+    public bool EngineIsRunning { get; set; }
+
+    internal void EngineStart()
+    {
+        if (!EngineIsRunning&&FillLevel>0)
+        {
+            EngineIsRunning = true;
+        }
+    }
+    private void EngineStop()
+    {
+        if (EngineIsRunning)
+        {
+            EngineIsRunning = false;
+        }
+    }
+
+    public string ReturnModel()
+    {
+        return Model;
+        //Console.WriteLine($"Your model is {Model}");
```

Comment:
please always verify that you do not have useless comments not related to your code in PR